sql_5.sql

```diff
@@ -0,0 +1,167 @@
+Uber
+
+CREATE TABLE address
```

Comment:
Whose address is this ? How this address table gets inserted or updated from a user perspective ?